### ISSUE TITLE
db.time read doesn't seem to work properly in 18.0.4

### ISSUE THREAD

**Issue by arj03** [ssbc/secure-scuttlebutt#208]
I was testing the performance of secure-scuttlebutt 18.0.4 to see if anything had changed after the flume upgrade and noticed that [test 6](https://github.com/ssbc/bench-ssb/blob/master/06-ssb-read.js) doesn't work anymore. It's the same used for createLogStream so it's quite easy to test with:

```
  pull(
    sbot.createLogStream({ keys: true, values: false }),
    pull.drain(function (msg) {
      console.log("got msg", msg)
    }, function() {
      console.log("done!")
    })
  )
```

Just returns "done!" after upgrading scuttlebot to secure-scuttlebutt 18.0.4.

**Comment by arj03** [comment]
This seems to be because I run on an "old" database?

**Comment by dominictarr** [comment]
if it's an old database, it should rebuild the indexes. I'm sure I remember change _all_ the versions when we added charwise, but I see 1 for the view versions. maybe this was caused by a bad merge? I saw this happen because of `links` though, not `time`.

Reminds me, I had an idea to use npm-shrinkwrap on a scuttlebot bundle release, and also publish each version as it's own module - so that multiple versions can be installed alongside each other. Then we could have tests that one database version is able to recover from another, and that one version is able to communicate with another, etc.
